lib/elixir/test/elixir/record_test.exs

```diff
@@ -127,6 +127,16 @@ defmodule RecordTest do
     refute match?(user(_: "other"), user())
   end
 
+  test "__record__/1" do
+    assert __record__(:timestamp) == [:date, :time]
```

Comment:
perhaps the `__record__` function would be more useful if it returns fields and default values:

```elixir
assert __record__(:timestamp) == [date: nil, time: nil]
```

I don't need such functionality at the moment, however.